core/mm/core_mmu.c

```diff
@@ -2504,6 +2520,7 @@ static void *phys_to_virt_tee_ram(paddr_t pa, size_t len)
 #else
 static void *phys_to_virt_tee_ram(paddr_t pa, size_t len)
 {
+	size_t page_count = ROUNDUP_DIV(len, SMALL_PAGE_SIZE);
```

Comment:
Could `pa` not be paged aligned? If so or for flexibility:
```c
	size_t page_count = (ROUNDUP(pa + len, SMALL_PAGE_SIZE) -
			     ROUNDDOWN(pa, SMALL_PAGE_SIZE)) / SMALL_PAGE_SIZE;
```
?

Reply:
Good catch, I'll update.